test_data/yam/agama_auto_leap.yaml

```diff
@@ -0,0 +1,7 @@
+---
+os_release_name: Leap
+guided_partitioning:
+  disks:
+    - vdb
+unused_disks:
+  - vda
```

Comment:
we are not doing any partitioning to have test data to validate that.

Reply:
Thanks, removed.